Source/PlayerEx.cpp

```diff
@@ -185,3 +210,28 @@ void PlayerEx::RecalculateValues()
 
 	playerArray[playerNum].boolRecalcValues = 1;
 }
+
+int PlayerEx::GetNumAvailableWorkers()					{ return isInited ? playerArray[playerNum].numAvailableWorkers : HFLNOTINITED;				}

```

Comment:
I think I would prefer to see this formatted as:

```C++
int PlayerEx::GetNumAvailableWorkers()
{
	return isInited ? playerArray[playerNum].numAvailableWorkers : HFLNOTINITED;
}
```

The ternary operator works for now on these simpler operations and saves some space. I wouldn't want to see it for more complex statements.

Unless others really want to see it spaced as this PR proposed, I would be willing to do the legwork post-merge to reformat.